frontend/src/components/cropper/index.js

```diff
@@ -98,42 +110,52 @@ class AvatarUpload extends Component {
             })
 
         this.setState({
-            inputAvatar: img
+            inputImage: img
         })
     }
 
-    saveAvatar = () => {
-        const avatar = this.state.croppedAvatar
+    saveImage = () => {
+        const image = this.state.croppedImage
+
         const fd = new FormData()
+        fd.append('file', image)
 
-        fd.append('file', avatar)
-
-        Axios.put(`${this.props.store.defaultData.backendUrl}/profile/picture`, 
-        fd, 
-        { withCredentials: true, 'content-type': 'multipart/form-data' })
-        .then((res) => {
-            this.props.changeAvatar(res.data.url) // put in Axios response
-                this.props.closeAvatarUpload()
-        }).catch(err => {
-            if (err.response.status === 401) {
-                this.props.history.push('/login/')
-            }
-        })
+        let address = this.props.store.defaultData.backendUrl
+        if (this.props.inputType === this.BANNER) {
+            address += '/profile/banner'
+        } else if (this.props.inputType === this.AVATAR) {
+            address += '/profile/picture'
+        }
```

Comment:
Option 1 (Better)
```suggestion
        const { inputType } = this.props
        let address = this.props.store.defaultData.backendUrl + '/profile/'
        
        if (inputType === this.BANNER) {
            address += 'banner'
        } else if (inputType === this.AVATAR) {
            address += 'picture'
        }
```

Would probably have been better to just use this.BANNER of this.AVATAR which would result in the following:

Option 2 (I think this is even better)
```suggestion
        const { inputType } = this.props
        let address = this.props.store.defaultData.backendUrl + '/profile/' + inputType
```